api/hiveextension/v1beta1/agentclusterinstall_types.go

```diff
@@ -317,7 +321,7 @@ type Networking struct {
 	NetworkType string `json:"networkType,omitempty"`
 
 	// UserManagedNetworking indicates if the networking is managed by the user.
-	// For single-node installations, set to true or leave empty.
+	// For single-node installations (none or external platfrom), set to true or leave empty.
```

Comment:
typo: platfrom

I believe that userManagedNetworking actually needs to be true if _any_ of {SNO, platform none, platform external} are true.
There's also some code to handle the case where it's left empty that will eventually need to be updated to include the external platform type: https://github.com/openshift/assisted-service/blob/master/pkg/webhooks/hiveextension/v1beta1/agentclusterinstall_mutating_hook.go#L158-L190 (for now it's not used in the agent installer though).